public/app/features/search/service/sql.ts

```diff
@@ -81,7 +81,10 @@ export class SQLSearcher implements GrafanaSearcher {
     }
 
     const limit = query.limit ?? (query.from !== undefined ? 1 : DEFAULT_MAX_VALUES);
-    const page = query.from !== undefined ? query.from / limit : undefined;
+    const page =
+      query.from !== undefined
+        ? query.from / limit + 1 // pages are 1-indexed
+        : undefined;
```

Comment:
```suggestion
    const page =
      query.from !== undefined
        ? (query.from / limit) + 1 // pages are 1-indexed
        : undefined;
```

can we bracket this or does prettier just undo it? reads much cleaner with the brackets imo

Reply:
I agree, but prettier undoes it. I'll try a prettier-ignore comment

Reply:
Looking again, I still don't think this math is correct 🤦🏻